src/numberApp.js

```diff
@@ -0,0 +1,55 @@
+import numberGame from './numberGame.js';
+
+const button = document.getElementById('guessButton');
+const numberInput = document.getElementById('input');
+const trysLeft = document.getElementById('trysLeft');
+const feedback = document.getElementById('feedback');
+const winLose = document.getElementById('winLose');
+let trys = 4;
+
+
+//random integer syntax pulled from MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomInt(max) {
+    return Math.floor(Math.random() * (max - 1)) + 1; 
+}
+let randomNum = getRandomInt(20); 
+
+
+button.addEventListener('click', () => {
+    trysLeft.textContent = 'Trys Left ' + trys;
```

Comment:
this first line inside the event listener could most likely be removed without worry.